model/pod/dispatcher/dispatcher.go

```diff
@@ -251,7 +251,7 @@ func (pd *PodDispatcher) checkTaskIsDispatchable(ctx context.Context, env evergr
 		return false, nil
 	}
 
-	refs, err := model.FindProjectRefsByIds(t.Project)
+	refs, err := model.FindMergedProjectRefsByIds(t.Project)
```

Comment:
Side thought is that this should've just called the regular FindMergedProjectRef since it's just for one project? Could ticket that though if you don't want to change it here

Reply:
Let's get 'er done 🤠